src/amo/components/HomeHeroBanner/index.js

```diff
@@ -116,12 +114,41 @@ export class HomeHeroBannerBase extends React.Component<Props> {
       ),
       (
         <HeroSection
-          key="tree-style-tab"
-          linkTo="/addon/tree-style-tab/"
+          key="hero-9"
+          linkTo="/addon/searchpreview/"
         >
-          <h3>{i18n.gettext('Tree Style Tab')}</h3>
+          <h3>{i18n.gettext('SearchPreview')}</h3>
 
-          <p>{i18n.gettext('Display tabs in a space-saving “tree” layout.')}</p>
+          <p>
+            {i18n.gettext(`Enhance search results with thumbnail previews,
+              popularity ranks & more.`)}
```

Comment:
I assume this `&` gets escaped by React, right? I think so.

Reply:
Seems to be the case. It displays fine when that item is the large item with the description displayed.